docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst

```diff
@@ -255,25 +227,24 @@ Portable Globe and Server 5.2
       You can view the KML of the shape you have drawn by selecting the
       **Enter polygon as KML** checkbox.
 
-      .. _Globe_Resolution:
-      .. rubric:: Globe resolution
+      .. _Portable_Resolution:
+      .. rubric:: Portable Resolution
 
-      The polygon you specified in the previous step defines your 'area of
-      interest.' This area will contain high-resolution imagery and
-      data, and search tabs will be created for information that lies
-      within this zone. The maximum and minimum resolutions are
-      specified as integers between 1 and 24. These correspond to the
-      zoom levels that are used in the Fusion server.
+      The polygon made in the previous step defines your *area of
+      interest*. This area will contain high-resolution content, and search
+      tabs will be created for information that lying within. The maximum and
+      minimum resolutions are specified as whole numbers between 1 and 24. These
+      correspond to the zoom levels used in GEE Fusion.
 
-      .. rubric:: Minimum resolution
+      .. rubric:: Minimum Resolution
 
       The zoom level for the polygon area is specified in the **Maximum

```

Comment:
Again, if this is now the Spartan view Cutter, I think this is the world level resolution?